### ISSUE TITLE
Bug when using sync

### ISSUE THREAD

**Issue by marianozicavo** [locize/locize-cli#14]
I'm running `locize sync --api-key myApiKeyHere --project-id myProjectIdHere` and getting the following error.
My locize project is brand new and has a single version called `latest`. It has two languages: `es`(reference) and `en`. 

I'm running it from a directory that has 2 directories inside: `es` and `end`. Each directory has a file for each namespace called, for example `Login.json`.

```
/usr/local/lib/node_modules/locize-cli/sync.js:436
    opt.isPrivate = downloads[0].isPrivate;
                                 ^

TypeError: Cannot read property 'isPrivate' of undefined
    at getDownloads (/usr/local/lib/node_modules/locize-cli/sync.js:436:34)
    at Request.request [as _callback] (/usr/local/lib/node_modules/locize-cli/sync.js:213:5)
    at Request.self.callback (/usr/local/lib/node_modules/locize-cli/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/usr/local/lib/node_modules/locize-cli/node_modules/request/request.js:1161:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/locize-cli/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:313:30)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)
```

Is there something I'm missing? Do I need to add an extra parameter to the command?

**Comment by jamuhl** [comment]
we're on it...found the cause already -> will update the code and publish an update in the next hours

**Comment by adrai** [comment]
should be fixed with v4.3.2

**Comment by marianozicavo** [comment]
It is! Thank you @jamuhl  and @adrai
